File: Aula06/exercicios/ex07.cpp
#include <iostream>
using namespace std;

string classificarNota(int nota){
    if(nota >= 90) return "A";
    else if(nota >= 80) return "B";
    else if(nota >= 70) return "C";
    else if(nota >= 60) return "D";
    else return "E";
}

int main(){
    int nota;
    cin >> nota;
    cout << classificarNota(nota) << endl;
}

File: Aula06/exercicios/ex07.py
nota = int(input("Nota: "))

if nota >= 90:
    print("A")
elif nota >= 80:
    print("B")
elif nota >= 70:
    print("C")
elif nota >= 60:
    print("D")
else:
    print("E")

def classificar(nota):
    if nota >= 90: return "A"
    elif nota >= 80: return "B"
    elif nota >= 70: return "C"
    elif nota >= 60: return "D"
    else: return "E"

print(classificar(nota))